File: languages/java/src/main/java/com/bloock/sdk/entity/key/ImportCertificateParams.java
package com.bloock.sdk.entity.key;

/**
 * Represents the parameters for importing a certificate.
 */
public class ImportCertificateParams {
  String password;

  /**
   * Constructs a new ImportCertificateParams object with the specified password.
   * @param password
   */
  public ImportCertificateParams(String password) {
    this.password = password;
  }

  /**
   * Constructs a new ImportCertificateParams object with no specified password.
   */
  public ImportCertificateParams() {}

  /**
   * Gets the password associated with the certificate.
   * @return
   */
  public String getPassword() {
    return password;
  }
}

File: languages/python/bloock/entity/key/import_certificate_params.py
from __future__ import annotations


class ImportCertificateParams:
    """
    Represents the parameters for importing a certificate.
    """
    password = None

    def __init__(self, password=None) -> None:
        """
        Constructs a new ImportCertificateParams object with the specified password.
        :type password: object
        :rtype: object
        """
        self.password = password
